model-server/src/main/kotlin/org/modelix/model/server/store/IgniteStoreClient.kt

```diff
@@ -103,6 +112,43 @@ class IgniteStoreClient(jdbcConfFile: File? = null, inmemory: Boolean = false) :
         return cache.associate { it.key to it.value }
     }
 
+    override fun removeRepositoryObjects(repositoryId: RepositoryId) {
+        if (!inmemory) {
+            removeRepositoryObjectsFromDatabase(repositoryId)
+        }
```

Comment:
It would probably be good to document here why deletion happens twice (when not in in-memory mode) and also why this order of operations is safe (why doesn't ignite try to access the keys that are already deleted now?)

Reply:
done